src/InteractiveWindow/Editor/Commands/InteractiveWindowCommands.cs

```diff
@@ -271,6 +278,20 @@ public void DisplayHelp()
                 _window.Write(HelpIndent);
                 _window.WriteLine(line);
             }
+
+            // Hack: Display script directives only in CS/VB interactive window
+            // TODO: https://github.com/dotnet/roslyn/issues/6441
+            var evaluatorTypeName = _window.Evaluator.GetType().Name;
```

Comment:
Consider using ```_window.Evaluator.LanguageName```, rather than the type.  Or if we really want it to only work for our types, consider using the full name.